lib/common.py

```diff
@@ -16,6 +17,17 @@ def vm_image(vm_key):
         url = DEF_VM_URL + url
     return url
 
+def objects_name_prefix():
+    try:
+        from data import OBJECTS_NAME_PREFIX
+        name_prefix = OBJECTS_NAME_PREFIX
+    except ImportError:
+        name_prefix = f"[{os.getlogin()}]"
+        if not name_prefix:
+            name_prefix = '[TEST]'
```

Comment:
Is it possible that name_prefix is not defined at this stage?

Reply:
i don't know, but in case we didn't catch a user name, it will be cover.

Reply:
The docs for os.getlogin or getpass.getuser don't mention a possible empty value, and I believe that if they really can't get the user name, they'll raise an Exception anyway.